### PR TITLE
Is this the right log loss function?

### PR DESCRIPTION
**Pull request by HanzhangRen**
Seems to be different from p.74 of https://www.slideshare.net/ShangxuanZhang/xgboost.

### CODE REVIEW

**Review by trivialfis — CHANGES_REQUESTED** [review]
The one in doc and the one in PPT are the same except for the sign.  Also, I think you have missed 2 minus signs in the rightmost log function.
